android/quest/src/test/resources/configs/ecbis-saa/config_forms.json

```diff
@@ -5,12 +5,18 @@
     {
       "form": "anc-patient-registration",
       "title": "Enroll as ANC Patient",
-      "identifier": "1924"
+      "identifier": "1924",
+      "saveButtonText": "Mark as ANC client",
+      "setPractitionerDetails" : true,
+      "setOrganizationDetails" : true
     },
     {
       "form": "family-member-registration",
       "title": "Add Family Member",
-      "identifier": "82953"
+      "identifier": "82953",
+      "saveButtonText": "Update Family Member",
```

Comment:
Shouldn't the `saveButtonText` read **ADD FAMILY MEMBER**? in caps as in the figma design.

Reply:
Are we using different questionnaires when updating member's details?